Dockerfile

```diff
@@ -36,6 +36,9 @@ RUN mkdir -m 0750 /root/.android
 ADD docker/adb/insecure_shared_adbkey /root/.android/adbkey
 ADD docker/adb/insecure_shared_adbkey.pub /root/.android/adbkey.pub
 
+# Allow all users to run "sudo tc"
+RUN echo 'ALL ALL=NOPASSWD: /usr/sbin/tc' > /etc/sudoers.d/tc
```

Comment:
I wasn't sure if this lines should go here or in https://github.com/sitespeedio/docker-browsers/blob/master/Dockerfile

Reply:
I think you are right, I can move it then before next release!

Reply:
We also use `route` and `ip`  with sudo see https://github.com/sitespeedio/throttle/blob/master/lib/tc.js when we use Throttle inside the container.  You can test that scenario using:
`docker run --cap-add=NET_ADMIN --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io https://www.sitespeed.io/ -b chrome -n 1 -c 3g --browsertime.connectivity.engine=throttle`

If also add those, we should be good to go!

Reply:
Okay, done in f039eb92f93ff097a3bee4cd83436be7d609fa14. The test script and output in the PR description have been updated to include that command.